app/modules/wifi/wifi_model.php

```diff
@@ -0,0 +1,127 @@
+<?php
+class wifi_model extends Model {
+
+	function __construct($serial='')
+	{
+		parent::__construct('id', 'wifi'); //primary key, tablename
+		$this->rs['id'] = 0;
+		$this->rs['serial_number'] = $serial; $this->rt['serial_number'] = 'VARCHAR(255) UNIQUE';
+		$this->rs['agrctlrssi'] = '';
```

Comment:
Please define the columns that are numeric as a number (this will aid with sorting/searching).

    $this->rs['agrctlrssi'] = 0;